### PR TITLE
fix test, moving test data to liquidbase

### PR DESCRIPTION
**Pull request by maiorovvv**

### CODE REVIEW

File: src/test/resources/application-test.properties [hunk 1]
```diff
@@ -11,6 +11,8 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
 spring.liquibase.contexts=test
 
+#spring.sql.init.schema-locations=classpath*:sql/schema.sql
```
Comment (Igor-Klein-1988) [inline src/test/resources/application-test.properties]: do you use this?

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -36,4 +36,8 @@
     <include file="changes/202402021110-insert-wishlist-dev.xml" relativeToChangelogFile="true" context="@dev"/>
     <include file="changes/202402081000-wishlist-item-table.xml" relativeToChangelogFile="true"/>
     <include file="changes/202402081010-insert-wishlist-item-table-dev.xml" relativeToChangelogFile="true" context="@dev"/>
+    <include file="changes/202402031338-token-table.xml" relativeToChangelogFile="true"/>
```
Comment (Igor-Klein-1988) [inline src/main/resources/db/changelog/db.changelog-master.xml:39]: Have you got `202402031338-token-table.xml` file?

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 3]
```diff
@@ -36,4 +36,8 @@
...
+    <include file="changes/20240215225600-insert-accounts-test.xml" relativeToChangelogFile="true" context="@test"/>
```
Comment (Igor-Klein-1988) [inline src/main/resources/db/changelog/db.changelog-master.xml]: incorrect name `20240215225600`

File: src/main/resources/db/changelog/db.changelog-master.xml (hunk 2 above)
Comment (maiorovvv) [reply src/main/resources/db/changelog/db.changelog-master.xml:39]: Of course there is, and my colleague Igor forgot to include it in this file.

**Review by Igor-Klein-1988 — APPROVED** [review]

**Pull request merged**